### PR TITLE
Us/be/valgrind support

### PR DESCRIPTION
**Pull request by Dogukan-lab**
The engine now supports building from WSL Ubuntu,  no clue if it works on Linux Native

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿# Blocky Engine
+﻿e# Blocky Engine
```
Comment (LarsingDash) [inline README.md]: hmmm

**Review by LarsingDash — CHANGES_REQUESTED** [review]
Fix readme rq then yes

**Comment by LarsingDash** [comment]
Misschien btw ook Tests standaard OFF zetten?

File: README.md (hunk 1 above)
Comment (Dogukan-lab) [reply README.md]: huh.

**Review by LarsingDash — APPROVED** [review]
Ye

**Pull request merged**